src/unix/linux_like/emscripten/mod.rs

```diff
@@ -777,9 +746,9 @@ pub const POSIX_MADV_WILLNEED: ::c_int = 3;
 
 pub const AT_EACCESS: ::c_int = 0x200;
 
-pub const S_IEXEC: mode_t = 0o0100;
-pub const S_IWRITE: mode_t = 0o0200;
-pub const S_IREAD: mode_t = 0o0400;
+pub const S_IEXEC: mode_t = 64;
+pub const S_IWRITE: mode_t = 128;
+pub const S_IREAD: mode_t = 256;
```

Comment:
These look like they are octal in the musl source, could they stay that way?

Reply:
I reverted this part, I can't remember why I did this in the first place.